db/schema.rb

```diff
@@ -29,8 +29,21 @@
     t.index ["category_id"], name: "index_items_on_category_id"
   end
 
+  create_table "order_items", force: :cascade do |t|
+    t.bigint "order_id"
+    t.bigint "item_id"
+    t.integer "cost", null: false
```

Comment:
might be worth nothing it's a unit_price for the original price of one item purchased